recipes/quazip/all/test_package/conanfile.py

```diff
@@ -0,0 +1,18 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if not tools.cross_building(self.settings):
+            zipFile_path = os.path.abspath(os.path.join(__file__, "..","zipFile.zip"))
+            bin_path = os.path.join("bin", "test_package")
+            self.run(bin_path + " " + zipFile_path, run_environment=True)
```

Comment:
```suggestion
		     open("zipfile.zip", "w")
            bin_path = os.path.join("bin", "test_package")
            self.run(bin_path + " zipfile.zip", run_environment=True)
```

You don't need an empty file in the repository.

Reply:
https://github.com/conan-io/conan-center-index/pull/4562/files#diff-9db0766669b7d377d7b1d76b995d978cfe449b16903d8c6229ee688c8d45eb1c It's marked as empty file by Github.

Reply:
I don't know why it says that.
If you follow that link and then click on the 3 dots, select view file and then view raw, you can see it is an actual zip file with 2 txts in it.

Reply:
understood.